codegen/config.python.json

```diff
@@ -0,0 +1,5 @@
+{
+	"packageName": "asposeimagingcloud",
+	"projectName": "aspose-imaging-cloud",
+	"packageVersion" : "1.0.0"
```

Comment:
@s-zubov Please, set the appropriate package version.

Reply:
Thank you, I've updated package version.